app/models/ontology_version.rb

```diff
@@ -1,16 +1,19 @@
 class OntologyVersion < ActiveRecord::Base
   include CodeReferencable
+  include Numbering
 
   include OntologyVersion::Files
   include OntologyVersion::States
   include OntologyVersion::Parsing
-  include OntologyVersion::Numbers
+  include OntologyVersion::Proving
   include OntologyVersion::OopsRequests
   include IRIUrlBuilder::Includeable
 
   include Rails.application.routes.url_helpers
   include ActionDispatch::Routing::UrlFor
 
+  numbering_parent_column 'ontology_id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.